packages/shared-lib/src/core/deviceConfigManifest.ts

```diff
@@ -154,7 +154,7 @@ export interface TableConfigManifestEntry extends ConfigManifestEntryBase {
 	defaultType: string
 	/** Used when the .config indexes are different from the type enum */
 	deviceTypesMapping?: any
-	/** The name of the the property used to decide the type of the entry */
+	/** The name the property used to decide the type of the entry */
```

Comment:
I think this was supposed to be `The name of the property`

Reply:
Correct, sloppy of me! Fixing.